src/rime/context.cc

```diff
@@ -121,6 +121,20 @@ bool Context::Select(size_t index) {
   return false;
 }
 
+bool Context::Hilite(size_t index) {
```

Comment:
Let's spell it `Highlight`.

It's one of the option I suggested in Leo's PR.
I would probably took his PR with some modifications.